operatoringress/v1/types.go

```diff
@@ -55,6 +55,14 @@ type DNSRecordSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +required
 	RecordTTL int64 `json:"recordTTL"`
+	// dnsManagementPolicy denotes the current policy applied on the dns record.
+	// Records that have policy set as "Unmanaged" are ignored by the ingress
+	// operator and should be managed at the discretion of the cluster admin.
```

Comment:
The doc on this type explicitly says: 

> Cluster admin manipulation of this resource is not supported.

Why are we telling them to manage a thing based on this?

Reply:
Basically there is no management that the cluster admins can do other than to delete the *DNSRecord* because once `dnsManagementPolicy` is set to `Unmanaged` the controller will ignore the resource silently and not perform any actions on it until the IC is updated back to `Managed` (will update the docs on this type). 

Maybe instead of telling
`should be managed at the discretion of the cluster admin`,
we could say
`the resource can be deleted at the discretion of the cluster admin`. 
WDYT?